federation/cmd/genfeddocs/gen_fed_docs.go

```diff
@@ -57,6 +61,11 @@ func main() {
 		// generate docs for kube-controller-manager
 		controllermanager := fedcmapp.NewControllerManagerCommand()
 		doc.GenMarkdownTree(controllermanager, outDir)
+	case "kubefed":
+		// generate docs for kubefed
+		defaultServerImage := fmt.Sprintf("%s:%s", kubefedapp.HyperkubeImageName, version.Get())
```

Comment:
I think it would be preferable to compose the default server image in a function and reuse it in both places.